client/src/pages/Homepage/Homepage.css

```diff
@@ -2,23 +2,70 @@ html, body {
     height: 100%;
 }
 
+.container-fluid {
+    padding: 80px 120px;
+}
+
 #content {
     height: 100%;
     width: 100%;
     background: linear-gradient(90deg, rgba(255, 255, 255, 0.84) 47%, #7799A4 53%);
```

Comment:
Can we move the background from the page content to just the first div?

Reply:
Minor suggestion: We can add ``` min-width: 100vw ``` to the parallax class to get rid of the white space on the right hand side of the div